services/multiinstance/staticdelay/shouldattest.go

```diff
@@ -60,12 +75,25 @@ func (s *Service) ShouldAttest(ctx context.Context, duty *attester.Duty) bool {
 		}
 
 		for _, attestation := range resp.Data {
-			if slices.ContainsFunc(validatorCommitteeIndices, func(index uint64) bool { return attestation.AggregationBits.BitAt(index) }) {
-				// An attestation is already in the pool; we don't need to act.
-				log.Trace().Msg("Another instance is attesting; not activating attester")
-				s.disableAttester(ctx)
+			// We only accept single-committee attestations.
+			if _, err := attestation.CommitteeIndex(); err != nil {
+				continue
+			}
+
+			aggregationBits, err := attestation.AggregationBits()
+			if err != nil {
+				log.Warn().Err(err).Msg("Failed to obtain aggregation bits")
+				continue
+			}
+			for validatorCommitteeIndex := range dutyAttestations {
+				if aggregationBits.BitAt(validatorCommitteeIndex) {
+					// if slices.ContainsFunc(validatorCommitteeIndices, func(index uint64) bool { return attestation.AggregationBits.BitAt(index) }) {
```

Comment:
Do we need this code?

Reply:
It's "modern go" syntactic sugar for the dual loop above it, but honestly I struggled with it.  It doesn't seem simpler or cleaner than the dual loop.  I'll remove it.